### DISCUSSION TITLE
3.0测试

### DISCUSSION THREAD

**Discussion by wsxh2333** [Melledy/Grasscutter#1518] [Q&A] [answered]
有3.0测试的物品id吗

**Comment by yzw-zero** [comment]
How to upgrade to 3.0?

**Reply by 4Benj** [reply]
There are unofficial versions of GC which support beta versions but the public version of GC does not (and possibly will never) support beta versions of the game. This is due to the fact that the company behind the game may become more hostile towards GC if we publicly support beta versions, additionally sometimes the company may make changes to the game which attempt to kill GC outright and those versions of GC have to accommodate for those changes which can take a lot of work, and if the company figures out how they fixed some of those changes before release, it can make the work that people have put into those versions possibly useless and may delay the public version of GC's support for the new versions.

**Comment by heixinduya** [comment]
怎么下载

**Comment by memetrollsXD** [comment, marked as answer]
Please read the following answer from 4BenJ: https://github.com/Grasscutters/Grasscutter/discussions/1518#discussioncomment-3164595
